rqpy/plotting/_iv_didv_tools_plotting.py

```diff
@@ -257,3 +323,455 @@ def _plot_energy_res_vs_bias(r0s,
+    -------
+    None
+
+    """
+
+    noise = data.noise_model
+    if idx == 'all':
+        inds = data.traninds
+    else:
+        inds = idx
+
+    for ind in inds:
+        if idx == 'all':
+            ii = ind - data.traninds[0]
+        else:
+            ii = ind - idx[0]
+        noise_row = data.df[data.noiseinds].iloc[ind]
+        r0 = noise_row.r0
+        f = noise_row.f
+        psd = noise_row.psd
+        didvobj = noise_row.didvobj_p
+
+        freqs = f[1:]
+        psd = psd[1:]
+
+        fig, ax = plt.subplots(1,1, figsize=(11,6))
+        if ylims_current is not None:
+            ax.set_ylim(ylims_current)
+        if xlims is not None:
+            ax.set_xlim(xlims)
+
+
+        ax.grid(which="major", linestyle='--')
+        ax.grid(which="minor", linestyle="dotted", alpha=0.5)
+        ax.tick_params(which="both", direction="in", right=True, top=True)
+        ax.set_xlabel(r'Frequency [Hz]')
+
+        ax.set_title(f"Current Noise For $R_0$ : {r0*1e3:.2f} $m\Omega$")
+        ax.loglog(
+            freqs,
+            np.sqrt(np.abs(data.noise_model['ites'][0][ii])),
+            color='#1f77b4',
+            linewidth=1.5,
+            label='TES Johnson Noise',
+        )
+        ax.fill_between(
+            freqs,
+            np.sqrt(np.abs(data.noise_model['ites'][1][ii])),
+            np.sqrt(np.abs(data.noise_model['ites'][2][ii])),
+            alpha=.5,
+            color='#1f77b4',
+        )
+        ax.loglog(
+            freqs,
+            np.sqrt(np.abs(data.noise_model['iload'][0][ii])),
+            color='#ff7f0e',
+            linewidth=1.5,
+            label='Load Noise',
+        )
+        ax.fill_between(
+            freqs,
+            np.sqrt(np.abs(data.noise_model['iload'][1][ii])),
+            np.sqrt(np.abs(data.noise_model['iload'][2][ii])),
+            alpha=.5,
+            color='#ff7f0e',
+        )
+        ax.loglog(
+            freqs,
+            np.sqrt(np.abs(data.noise_model['itfn'][0][ii])),
+            color='#2ca02c',
+            linewidth=1.5,
+            label='TFN Noise'
+        )
+        ax.fill_between(
+            freqs,
+            np.sqrt(np.abs(data.noise_model['itfn'][1][ii])),
+            np.sqrt(np.abs(data.noise_model['itfn'][2][ii])),
+            alpha=.5,
+            color='#2ca02c',
+        )
+        ax.loglog(
+            freqs,
+            np.sqrt(np.abs(data.noise_model['itot'][0][ii])),
+            color='#d62728',
+            linewidth=1.5,
+            label='Total Noise',
+        )
+        ax.fill_between(
+            freqs,
+            np.sqrt(np.abs(data.noise_model['itot'][1][ii])),
+            np.sqrt(np.abs(data.noise_model['itot'][2][ii])),
+            alpha=.5,
+            color='#d62728',
+        )
+        ax.loglog(
+            freqs,
+            np.sqrt(np.abs(data.noise_model['isquid'][0][ii])),
+            color='#9467bd',
+            linewidth=1.5,
+            label='Squid+Electronics Noise',
```

Comment:
Perhaps we should replace the `+` with an ampersand `&`? I feel that the plus sign is a little colloquial for plots.
```suggestion
            label='Squid & Electronics Noise',
```

Reply:
added